pkg/pgmodel/cache/flags.go

```diff
@@ -45,12 +45,12 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	/* set defaults */
 	cfg.seriesCacheMemoryMaxFlag.SetPercent(50)
 
-	fs.Uint64Var(&cfg.MetricsCacheSize, "metrics-cache-size", DefaultMetricCacheSize, "Maximum number of metric names to cache.")
-	fs.Uint64Var(&cfg.SeriesCacheInitialSize, "series-cache-initial-size", DefaultSeriesCacheSize, "Maximum number of series to cache.")
-	fs.Uint64Var(&cfg.LabelsCacheSize, "labels-cache-size", 10000, "Maximum number of labels to cache.")
-	fs.Uint64Var(&cfg.ExemplarKeyPosCacheSize, "exemplar-cache-size", DefaultExemplarKeyPosCacheSize, "Maximum number of exemplar metrics key-position to cache. "+
+	fs.Uint64Var(&cfg.MetricsCacheSize, "metrics.cache.metrics.size", DefaultMetricCacheSize, "Maximum number of metric names to cache.")
```

Comment:
This is confusing. how about

```suggestion
	fs.Uint64Var(&cfg.MetricsCacheSize, "metrics.cache.metric-name.size", DefaultMetricCacheSize, "Maximum number of metric names to cache.")
```

Why not keep a segment as `cache.`, so this becomes `cache.metric-name.size`?